core/src/main/java/org/ehcache/spi/cache/tiering/LowerCachingTier.java

```diff
@@ -38,7 +38,7 @@
    * @return the value holder, or null.
    * @throws CacheAccessException
    */
-  Store.ValueHolder<V> getOrComputeIfAbsent(K key, Function<K, Store.ValueHolder<V>> source) throws CacheAccessException;
+  Store.ValueHolder<V> installMapping(K key, Function<K, Store.ValueHolder<V>> source) throws CacheAccessException;
```

Comment:
It is a `LowerCachingTier` method that has the exact same name as a `CachingTier` method but a completely different contract. Since I was touching code in the area I could not resist the rename that really should have been part of the PR introducing this interface. I will isolate in its own commit and reference that issue instead.